src/System.Management.Automation/engine/remoting/commands/getrunspacecommand.cs

```diff
@@ -341,8 +342,22 @@ public string CertificateThumbprint
         /// </summary>
         protected override void BeginProcessing()
         {
-            base.BeginProcessing();
+#if UNIX
+            if (ComputerName?.Length > 0)
```

Comment:
Our common pattern is to check whether a parameter presents.
```c#
if (MyInvocation.BoundParameters.ContainsKey(nameof(ComputerName)))
```

Reply:
Is there a particular reason for this. The code uses the `ComputerName` property value and thus it makes more sense to do the check on the actual property rather than a pwsh-ism here. It's also cleaner and shorter than accessing a dictionary to check if a key exists vs just checking the property value.